packages/contracts/scripts/task/dollar/MigrateMetapool.s.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "forge-std/Script.sol";
+
+import "src/dollar/interfaces/IMetaPool.sol";
+import "src/dollar/Staking.sol";
+import "../../../lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import "src/dollar/core/TWAPOracleDollar3pool.sol";
+import "src/dollar/core/UbiquityDollarManager.sol";
+
+contract migrateFunds is Script {
+	function run() public virtual {
+
+		
+		IMetaPool v2Metapool = IMetaPool(0x20955CB69Ae1515962177D164dfC9522feef567E);
+		IMetaPool v3Metapool = IMetaPool(0x9558b18f021FC3cBa1c9B777603829A42244818b);
+		IERC20 USD3CRVToken = IERC20(0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490);
+		IERC20 uAD = IERC20(0x0F644658510c95CB46955e55D7BA9DDa9E9fBEc6);
```

Comment:
I believe I banned `uAD` via cspell, perhaps I didn't merge it yet?

@0xcodercrane and I discussed making a code styleguide we'll have to make it soon here. 

Anyways consider calling this `UbiquityDollar` as its purpose is clearer for new developers. Also we have a token symbol setter in the contract so it is possible to update the token symbol after deployment, so `uAD` could be deprecated.
```suggestion
		IERC20 UbiquityDollar = IERC20(0x0F644658510c95CB46955e55D7BA9DDa9E9fBEc6);
```

Reply:
Done